_includes/guide/credentials.html

```diff
@@ -0,0 +1,84 @@
+<p><button class="accordion" type="button" aria-expanded="false"> Extra-Small ACO (50 synthetic beneficiaries) </button></p>
+<div class="acc_content">
+    Client ID:
+    {%- capture client_id -%}
+    3841c594-a8c0-41e5-98cc-38bb45360d3c
+    {%- endcapture -%}
+
+    {% include copy_snippet.md code=client_id %}
```

Comment:
@mtrang1263 This is the other accessibility issue I called out in Slack - we need appropriate alt text for the copy to clipboard buttons.

Reply:
Leveraging `aria-label` to leverage more rich context in the Copy to Clipboard button